packages/hierarchies-react/src/presentation-hierarchies-react/stratakit/TreeAction.tsx

```diff
@@ -0,0 +1,62 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+import { ComponentProps, memo } from "react";
+import { DropdownMenu, Tree } from "@stratakit/structures";
+
+/**
+ * Attributes used to decide how to render tree actions. They allow to differentiate action style based on the context
+ * where that action is used.
```

Comment:
Let's call this `StrataKitTreeActionProps` to be consistent with other type names we have.